.chloggen/extension-memory-limiter.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: extension/memoryLimiter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: move core processor/memorylimiter -> internal/memorylimiter and use in both extension/memorylimiter and processor/memorylimite
```

This is a user-facing change log. No need to highlight implementation detatils
```suggestion
note: Introduce a `memory_limiter` extension which receivers can use to reject incoming requests when collector doesn't have enough memory
```